erpnext/controllers/stock_controller.py

```diff
@@ -19,7 +19,8 @@ class QualityInspectionNotSubmittedError(frappe.ValidationError): pass
 class StockController(AccountsController):
 	def validate(self):
 		super(StockController, self).validate()
-		self.validate_inspection()
+		if not self.is_return:
```

Comment:
![image](https://user-images.githubusercontent.com/42651287/84425087-a38b8100-ac3e-11ea-9cf3-25847256cad8.png)

Many doctypes are inherited from Stock Controller and some of them don't have the is return field. 
```suggestion
		if not self.get('is_return'):
```